src/core_modules/capture-core/components/Pages/ViewEvent/epics/viewEvent.epics.js

```diff
@@ -87,9 +88,16 @@ export const getOrgUnitOnUrlUpdateEpic = (action$: InputObservable) =>
         ofType(viewEventActionTypes.EVENT_FROM_URL_RETRIEVED),
         switchMap((action) => {
             const eventContainer = action.payload.eventContainer;
-            return getApi()
-                .get(`organisationUnits/${eventContainer.event.orgUnitId}`)
-                .then(orgUnit => orgUnitRetrievedOnUrlUpdate(orgUnit, eventContainer))
+            // change from organisationUnitGroups -> groups
+            return Promise.all(
+                [
+                    getApi().get(`organisationUnits/${eventContainer.event.orgUnitId}`),
+                    getAssociatedOrgUnitGroups(eventContainer.event.orgUnitId),
+                ])
+                .then(([orgUnit, groups]) => {
+                    orgUnit.groups = groups;
```

Comment:
We agreed to set `orgUnit.groups = orgUnit.organisationUnitGroups`, but `orgUnit.organisationUnitGroups` turned out to only contain orgUnitGroupIds. I figured this approach, using `getAssociatedOrgUnitGroups`, could work because another async function `getApi` was already being used here.

Reply:
That works 👍